src/client/ui/filter_stories.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+
+export default class FilterText extends React.Component {
+  onChange(event) {
+    const filterText = event.target.value;
+    this.props.onChange(filterText);
+  }
+
+  render() {
+    const mainStyle = {
+      border: '1px solid #C1C1C1',
+      borderRadius: '2px',
+    };
+
+    const filterTextWrapStyle = {
+      background: '#F7F7F7',
+      paddingRight: '24px',
+    };
+
+    const filterTextStyle = {
+      fontSize: '15px',
```

Comment:
@arunoda Ah, didn't knew react do that. Updated. Change the component name as well.